src/ripple/beast/core/LexicalCast.h

```diff
@@ -84,6 +84,9 @@ struct LexicalCast<Out, std::string>
         auto first = in.data();
         auto last = in.data() + in.size();
 
+        if (first != last && *first == '+')
```

Comment:
Have to check the next one is digit. See @HowardHinnant comment.

Reply:
I don't think so - the semantics seem correct to me. If it's a plus and the next character is not a digit, then `std::from_chars` will return some kind of failure and the check, which ensures that (a) not failure is returned; and (b) that the whole string was parsed and nothing was left over will succeed. 

Check out this [example](https://godbolt.org/z/TMfYoMvYE). As you can see, the only cases where the semantics *don't* match is when you have a numeric value prefixed with a `+` sign. The "new" code does what we want, and the old one just fails.